packages/patterns-react/src/patterns/LeadSpaceCentered/LeadSpaceCentered.js

```diff
@@ -98,12 +98,23 @@ const LeadSpaceCentered = ({
       <div className={overlayClassname(gradient)}>
         <div className={`${prefix}--leadspace--centered__content`}>
           <h1 className={`${prefix}--leadspace--centered__title`}>{title}</h1>
-          <p className={`${prefix}--leadspace--centered__desc`}>{copy}</p>
-          {buttons && buttons.length > 0 && <ButtonGroup buttons={buttons} />}
+          <p
+            data-autoid={`${stablePrefix}--leadspace--centered__desc`}
+            className={`${prefix}--leadspace--centered__desc`}>
+            {copy}
+          </p>
+          {buttons && buttons.length > 0 && (
+            <ButtonGroup
+              data-autoid={`${stablePrefix}--leadspace--centered__button-group`}
```

Comment:
I was going to do this a bit later but can you edit `ButtonGroup` to accept this prop? 😁 It doesn't actually set the data id here. Probably pass it in as `autoid` prop and then in `ButtonGroup` you set `data-autoid` there.

Reply:
The end goal is to add `autoid` to each button and it's parent.
It seems I have to work on ButtonGroup.

Reply:
Right, yeah you can pass in the base part of the autoid then.

```
<ButtonGroup autoid={`${stablePrefix}--leadspace--centered`) ..... />
```

and then in ButtonGroup.js:
```
const ButtonGroup = ({ buttons, autoid }) => {
  return featureFlag(
    DDS_BUTTON_GROUP,
    <ol
      className={`${prefix}--buttongroup`}
      data-autoid={`${autoid}--button-group`}>
      {buttons.map((button, key) => {
        return (
          <li key={key} className={`${prefix}--buttongroup-item`}>
            <Button
              data-autoid={`${autoid}--button-group-${key}`}
              {...button}
              kind={key === 0 ? 'primary' : 'tertiary'}>
              {button.copy}
            </Button>
          </li>
        );
      })}
    </ol>
  );
};
```